src/crawlee/storages/_key_value_store.py

```diff
@@ -175,3 +182,56 @@ async def get_public_url(self, key: str) -> str:
             The public URL for the given key.
         """
         return await self._resource_client.get_public_url(key)
+
+    async def get_auto_saved_value(self, key: str, default_value: dict[str, Any] | None = None) -> dict[str, Any]:
+        """Gets a value from store that will be automatically saved on changes.
```

Comment:
Could we use KVS in the docs please? `store` could be easily mistaken for something else

```suggestion
        """Gets a value from KVS that will be automatically saved on changes.
```